pom.xml

```diff
@@ -63,8 +63,8 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <rocksdb.version>6.22.1.1</rocksdb.version>
         <protobuf.version>3.5.1</protobuf.version>
-        <log4j.version>1.2.17</log4j.version>
-        <slf4j.version>1.7.16</slf4j.version>
+        <log4j.version>2.15</log4j.version>
```

Comment:
should it be `2.15.0`?

Reply:
It is recommended to upgrade to `2.17.1`, which fixes all known vulnerabilities, see: https://logging.apache.org/log4j/2.x/security.html.